contracts/RestrictedNORI.sol

```diff
@@ -982,10 +980,13 @@ contract RestrictedNORI is
     address account
   ) private view returns (uint256) {
     Schedule storage schedule = _scheduleIdToScheduleStruct[scheduleId];
-    uint256 scheduleTrueTotal = _scheduleTrueTotal(schedule, scheduleId);
+    uint256 scheduleTrueTotal = _scheduleTrueTotal(
```

Comment:
definitely looks like `_quantityToRevokePerTokenHolder` can be optimized (not in this pr). as this function currently appears to be performing several responsibilities (burning, sending bp nori, and revoking)